templates/release_checklist.md

```diff
@@ -0,0 +1,149 @@
+Thanks!
+```
+
+</p>
+</details>
+
+- [ ] Create WPiOS and WPAndroid builds from `develop`
+- [ ] [Perform general writing flow tests on Android](https://github.com/wordpress-mobile/test-cases/tree/master/test-cases/gutenberg/writing-flow).
+- [ ] [Perform general writing flow tests on iOS](https://github.com/wordpress-mobile/test-cases/tree/master/test-cases/gutenberg/writing-flow).
+
+<!-- /scheduled_release_only -->
+
+## Create the Release
+
+<!-- scheduled_release_only -->
+
+On {{release_date}}:
+
+<!-- /scheduled_release_only -->
+
+- [ ] [Verify Aztec is up to date](https://github.com/wordpress-mobile/release-toolkit-gutenberg-mobile/blob/develop/Releasing.md#handling-aztec-updates).
+- [ ] Run `./release_automation.sh` from the release toolkit to generate the main apps integration PRs [\*](#).
+- [ ] Notify team that the release has been cut.
+<details>
+<summary>Message Template</summary>
+<p>
+
+```markdown
+⚠️ The gutenberg-mobile {{version_number}} release branches are now cut.
+Please do not merge any Gutenberg-related changes into the WPiOS or WPAndroid
+develop branches until after the main apps cut their own releases next week.
+If you’d like to merge changes now, merge them into the `gutenberg/after_{{version_number}}` branches.
+```
+
+</p>
+</details>
+
+- [ ] [Verify localization string files have been generated properly](https://github.com/wordpress-mobile/release-toolkit-gutenberg-mobile/blob/develop/Releasing.md).
+- [ ] Update `Unreleased` section in [`RELEASE-NOTES.txt`](https://github.com/wordpress-mobile/gutenberg-mobile/blob/develop/RELEASE-NOTES.txt) and [`gutenberg/../CHANGELOG.md`](https://github.com/WordPress/gutenberg/blob/trunk/packages/react-native-editor/CHANGELOG.md). Add a new `Unreleased` section after updating the release notes.
+- [ ] Perform quick smoke test of the editor on both iOS and Android builds
+- [ ] Update [Gutenberg Mobile PR](https://github.com/wordpress-mobile/gutenberg-mobile/pulls?q=is%3Apr+is%3Aopen+label%3Arelease-process) description. Make sure the "Changes" section informs testers on what is in the release.
+- [ ] Mark all 4 PRs ready for review.
+
+<!-- non_scheduled_release_only -->
+
+- [ ] Notify the team members directly resposible for handling the releases of WPiOS and WPAndroid.
+
+<!-- /non_scheduled_release_only -->
+
+<!-- optional_aztec_release -->
+
+<!-- optional_incoming_changes -->
+
+## Integrate the Release
+
+<!-- scheduled_release_only -->
+
+On {{release_date}}:
+
+<!-- /scheduled_release_only -->
+
+- [ ] Verify `gutenberg` ref within the gutenberg-mobile release branch is pointing to the [latest commit](#) in Gutenberg release branch.
+- [ ] Create and push a `rnmobile/{{version_number}}` git tag for the head of [Gutenberg release branch](https://github.com/WordPress/gutenberg/branches/all?query=rnmobile%2Frelease).
+- [ ] [Update bundles in the Gutenberg Mobile release branch](#).
```

Comment:
As per the corresponding step from the original checklist, I don't think is mandatory to update the bundle files, so maybe we should update the message, wdyt?

> Ensure that the bundle files are updated to include any changes to the release branch by running npm run bundle in gutenberg-mobile release branch and committing any changes.